pyrisco/risco.py

```diff
@@ -36,41 +44,60 @@
 
 
 class Partition:
-    """A representation of a Risco partition."""
+    """A representation of a Risco partition (or panel in case of no partitions)."""
```

Comment:
I think this should be a different class in the no partition case, since basically all properties now have the same `if self._partitions_available:` pattern. They can have a common base.